### PR TITLE
Show Unlock Feature modal for Storage actions

### PR DESCRIPTION
**Pull request by alex-deaconu**
## Description
Show Unlock Feature modal for Storage actions

Update e2e tests

[stage-18]

## Motivation and Context
Promote subscription for Storage.

## How Has This Been Tested?
Tested changes locally; updated unit & e2e tests.

## Release Plan:
- As the Submitter, upon requesting review of this pull request, I confirm that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed. 
- As the Reviewer, upon approving the changes in this PR, I confirm I have reviewed and I agree that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed

#### Release Checklist Items Skipped?
No

### CODE REVIEW

**Comment by alex-deaconu** [comment]
@ezequielc second PR is ready for review. Thanks!

File: test/e2e/registration/cases/trial.js [hunk 1]
```diff
@@ -43,18 +43,17 @@ var TrialScenarios = function() {
...
-        expect(storageSelectorModalPage.getActiveTrialBanner().isDisplayed()).to.eventually.be.true;
-        expect(storageSelectorModalPage.getStartTrialButton().isDisplayed()).to.eventually.be.false;
-
-        expect(storageHomePage.getNewFolderButton().isDisplayed()).to.eventually.be.true;
+        expect(storageHomePage.getUploadButtonUnsubscribed().isDisplayed()).to.eventually.be.false;
+        expect(storageHomePage.getUploadFolderButtonUnsubscribed().isDisplayed()).to.eventually.be.false;
       });
 
-      it('should show Strage trial after page refresh', function () {
+      it('should show Upload buttons after page refresh', function () {
```
Comment (ezequielc) [inline test/e2e/registration/cases/trial.js:54]: @alex-deaconu Does it still make sense to test the page refresh here? I am wondering if we can speed up the test if the buttons are already enabled. Thanks!

**Review by ezequielc — APPROVED** [review]
@alex-deaconu Looks good 👍 One comment about e2e tests. Thanks!

File: test/e2e/registration/cases/trial.js (hunk 1 above)
Comment (alex-deaconu) [reply test/e2e/registration/cases/trial.js:54]: @ezequielc to confirm, the first test is for the Local license being applied. The second test is to verify that the Trial Subscription was initiated in Chargebee and Store/Core received the response. I would suggest leaving it in. Thanks.

Comment (ezequielc) [reply test/e2e/registration/cases/trial.js:54]: @alex-deaconu Aha, good point. I agree on leaving as is. Thanks!

**Pull request merged**